opencga-catalog/src/main/java/org/opencb/opencga/catalog/db/mongodb/MongoDBAdaptorFactory.java

```diff
@@ -179,6 +172,14 @@ public MongoDBAdaptorFactory(Configuration catalogConfiguration) throws CatalogD
             }
         }
 
+        MongoDBConfiguration mongoDBConfiguration = MongoDBConfiguration.builder()
+                .setUserPassword(
+                        catalogConfiguration.getCatalog().getDatabase().getUser(),
+                        catalogConfiguration.getCatalog().getDatabase().getPassword())
+                .setServerAddress(dataStoreServerAddresses)
+                .load(catalogConfiguration.getCatalog().getDatabase().getOptions())
+                .build();
+
         this.mongoManager = new MongoDataStoreManager(dataStoreServerAddresses);
```

Comment:
I see. This configuration is only used for obtaining the objects "MongoDataStore" from the "MongoDataStoreManager". Ideally, the "MongoDataStoreManager" would have a configuration object. This weird behavior could be because, in the past, the MongoDataStoreManager used to hold a static map of connections, and the object had some kind of "singleton" pattern.